auth/v1.js

```diff
@@ -134,14 +128,14 @@ api.declare({
 }, async function(req, res) {
   let clientId = req.params.clientId;
 
-  // Load client
-  let client = await this.Client.load({clientId}, true);
+  // Load client from the resolver's cache
+  let client = this.resolver.jsonClient(clientId);
```

Comment:
Hmm... Honestly, how bad can it be... If we do continuationtoken in the future loading shouldn't be a problem...

Maybe one day we'll fix azure-entities to support prefix matching...I had it working in a fork, but it requires full table migration as we change the keys...

BTW, usually the query operation on azure returns upwards 1k entries... Per request... So even 10k users should be okay... Especially if we do continuationtoken at some point..

But let's first get that 1k users :)